calyx-py/test/correctness/arbiter_6.py

```diff
@@ -106,59 +30,64 @@ def add_wrap2(prog):
     j_mod_4 = wrap.reg("j_mod_4", 32)
 
     # Additional cells and groups to compute equality and lt
-    eq0cell, eq0grp = add_eq(wrap, i, 0, "eq0", "i_eq_0")
-    eq1cell, eq1grp = add_eq(wrap, i, 1, "eq1", "i_eq_1")
-    lt1cell, lt1grp = add_lt(wrap, j, 4, "lt1", "j_lt_4")
-    lt2cell, lt2grp = add_lt(wrap, j, 8, "lt2", "j_lt_8")
+    i_eq_0_cell, i_eq_0_grp = util.insert_eq(wrap, i, 0, "i_eq_0", 32)
```

Comment:
I frequently find myself receiving from a helper the following tuple:
- a cell
- a combinational group
that together perform some computation.

Reply:
If [this](https://github.com/cucapra/calyx/pull/1630/files#r1271335901) is put into place, I'll be able to leave this tuple whole and just pass it along in the control section.